mappings/net/minecraft/entity/ai/brain/Brain.mapping

```diff
@@ -21,7 +21,8 @@ CLASS aiw net/minecraft/entity/ai/brain/Brain
 	METHOD b forget (Laod;)V
 	METHOD b setActivity (Lawe;)V
 	METHOD b stopAllTasks (Lvd;Laih;)V
+	METHOD c getMemory (Laod;)Ljava/util/Optional;
```

Comment:
What's the difference in implementation between `getMemory` and `getOptionalMemory`?

Reply:
`getMemory` is the most used merhod and will throw an exception if there is no memory. `getOptionalMemory` has just two usages, and will return Optional.empty() if it's missing.

Reply:
`getMemory` is the most used merhod and will throw an exception if there is no memory. `getOptionalMemory` has just two usages, and will return Optional.empty() if it's missing.

Reply:
`getMemory` also returns an `Optional`, though